tools/xl/xl_parse.c

```diff
@@ -1151,7 +1154,7 @@ static void parse_vkb_list(const XLU_Config *config,
 
             if (vkb->backend_type == LIBXL_VKB_BACKEND_UNKNOWN) {
                 fprintf(stderr, "backend-type should be set in vkb spec\n");
-                rc = -1; goto out;
+                rc = ERROR_FAIL; goto out;
```

Comment:
new line?

Reply:
Error code paths with an if statement and a goto or a return on the same
line are allowed.